src/components/common/string/StringField.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from "react";
+import { InputLabel } from "@madie/madie-design-system/dist/react/";
+import { FormControl } from "@mui/material";
+import { TextField } from "@madie/madie-design-system/dist/react";
+
+const labelStyle = {
+  color: "#125496",
+};
+
+export interface StringFieldProps {
+  label: string;
+  fieldValue: string;
+  canEdit: boolean;
+}
+
+const StringField = ({ label, fieldValue, canEdit }: StringFieldProps) => {
+  const [field, setField] = useState<string>(fieldValue);
+  return (
+    <div style={{ width: "134px" }}>
+      <FormControl>
+        <InputLabel
+          id="string-field-label"
+          data-testid="string-field-label"
+          required={false}
+          sx={labelStyle}
+        >
+          {label}
+        </InputLabel>
+        <TextField
+          disabled={!canEdit}
+          label={label}
```

Comment:
Do we need 2 labels with same name ? Also can you provide screenshot for the implementation. that way its easier to review.

Reply:
@RohitKandimalla Yes based on the design, it should be there (https://www.figma.com/file/1RPYLI39hfMczAL81ARArX/MADiE-Application?type=design&node-id=9969-361838&mode=design&t=LTibHEo7B0nfMluY-0)

Reply:
![Pasted Graphic 1](https://github.com/MeasureAuthoringTool/madie-patient/assets/97188375/113fd0be-3cd3-46e1-90ea-8408cfc21ac8)